opentracing/example/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+// This is a near wholesale copy of the opentracing-go/examples/dapperish.go
+// file. The only thing that has been changed is the addition of an Appdash
+// store, traceapp, and appdash/opentracing Tracer. Everything else is the
+// exact same to show that it changing backends is O(1).
+// One thing I've noticed is the clash of package names, since there is
+// opentracing/opentracing and appdash/opentracing
```

Comment:
- Keep examples in the top-level `examples` directory instead of here.
- Rename to `webapp-opentracing` for example.
- How do you feel about writing this example as a wholesale copy of `webapp` in https://github.com/sourcegraph/appdash/tree/master/examples/cmd instead? I have a lot of cleanup concerns about this code, otherwise. Thoughts?